frontend/packages/client/src/api/notificationService.js

```diff
@@ -13,24 +14,40 @@ const options = {
   headers: { accept: 'application/json' },
 };
 
-export const startLeanplum = () => {
+const getDesiredAttributes = (communitySubIntentions) => {
```

Comment:
This is to convert the array in line 18 into a format of {community1:'True', community2:'False'}